### DISCUSSION TITLE
BUG FEEDBACK

### DISCUSSION THREAD

**Discussion by Dorp-lab** [MintCoolMC/MCBE_RainbowPieUI2#29] [General]
Some textures in the game are duplicated and misplaced.And the remaining amount of fuel in the fuel tanks of furnaces, blast furnaces, and smoke furnaces, as well as their heat indicators, will only be updated when exiting and reopening the panels
![Screenshot_20240619_130923_com mojang minecraftpe](https://github.com/MintCoolMC/MCBE_RainbowPieUI2/assets/173231805/2fd5b2af-9330-4f17-b82f-1e436e52c018)
![Screenshot_20240619_130328_com mojang minecraftpe](https://github.com/MintCoolMC/MCBE_RainbowPieUI2/assets/173231805/d28983de-ff5f-49dd-a209-cc0b8fef5b52)
![Screenshot_20240619_124011_com mojang minecraftpe](https://github.com/MintCoolMC/MCBE_RainbowPieUI2/assets/173231805/d74da880-8e3c-402d-aba4-b4ccb1b48332)

**Comment by Dorp-lab** [comment]
I hope developers of Rainbow UI can quickly fix these issues，Thanks.

**Comment by MintCoolMC** [comment]
Have you activated different versions of the resource pack in global resource packs and world resource packs at the same time?

**Comment by Dorp-lab** [comment]
No,I'm not.The world resource package and global resource package both use the same version, it's just the CI versions for Minecraft&nbsp;Bedrock Edition 1.21.0&nbsp;.And&nbsp;I have found another bug where the player's shortcut bar cannot be displayed properly while playing on the server, and they cannot interact with the shortcut bar and backpack.I hope developers of Rainbow UI can quickly fix these issue.
Thanks.



---Original---
From: "Mintraspberry ***@***.***&gt;
Date: Wed, Jun 19, 2024 19:23 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [MintCoolMC/MCBE_RainbowPieUI2] BUG FEEDBACK (Discussion #29)




 
Have you activated different versions of the resource pack in global resource packs and world resource packs at the same time?

**Reply by MintCoolMC** [reply]
I can't read this, use your native language. Thanks

**Comment by Dorp-lab** [comment]
总结一下，我发现的所有bug
第一个游戏内部分贴图错位和重复（资源包用的是最新的版本，不过是ci版本，Minecraft版本为1.21.0）
第二个是服务器内游玩时无法正确显示快捷栏，并且无法与快捷栏和背包进行交互（服主没有把资源包ban掉，服务器正常）
第三个问题，就是熔炉高炉和烟熏炉的热度指示器以及燃料槽中的燃料数量，只有在退出并重新打开面板时才会更新
希望作者能够快速解决问题
麻烦了，谢谢(*^o^*)



---Original---
From: "Mintraspberry ***@***.***&gt;
Date: Wed, Jun 19, 2024 19:44 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [MintCoolMC/MCBE_RainbowPieUI2] BUG FEEDBACK (Discussion #29)




 
I can't read this, use your native language. Thanks

**Comment by MintCoolMC** [comment]
根据你提供的信息，你应该是在全局资源包中使用了ci版本，在世界资源包中使用了2.5.10126版本，从而导致了UI内容重复，解决方法很简单，把世界资源包也更新或者取消在世界资源包启用UI

**Comment by Dorp-lab** [comment]
谢谢作者(*^o^*)



---Original---
From: "Mintraspberry ***@***.***&gt;
Date: Wed, Jun 19, 2024 20:08 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [MintCoolMC/MCBE_RainbowPieUI2] BUG FEEDBACK (Discussion #29)




 
根据你提供的信息，你应该是在全局资源包中使用了ci版本，在世界资源包中使用了2.5.10126版本，从而导致了UI内容重复，解决方法很简单，把世界资源包也更新或者取消在世界资源包启用UI

**Comment by Dorp-lab** [comment]
作者大大，我这又发现一个问题
![Screenshot_20240619_214132_com mojang minecraftpe](https://github.com/MintCoolMC/MCBE_RainbowPieUI2/assets/173231805/ce18118a-68a8-437d-a77c-67b1c6fd0d8c)
是怎么回事？要重下吗？

**Reply by MintCoolMC** [reply]
刪掉舊版下載最新版

**Comment by Dorp-lab** [comment]
明白了，谢谢作者



---Original---
From: "Mintraspberry ***@***.***&gt;
Date: Thu, Jun 20, 2024 02:40 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [MintCoolMC/MCBE_RainbowPieUI2] BUG FEEDBACK (Discussion #29)




 
刪掉舊版下載最新版
